.changeset/poor-bees-pull.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap/guided-answers-extension-core': minor
```

Comment:
I don't see changes in `@sap/guided-answers-extension-core`, I think we can skip this
```suggestion
```

Reply:
I don't know why is picking it up? I can remove it
![Screenshot 2022-06-23 at 10 54 06](https://user-images.githubusercontent.com/2386570/175272074-1c9dad94-fbc1-4b03-8e31-4c6657377afb.png)

Reply:
No idea, perhaps you need to merge latest `main` branch? In file list of this PR nothing changed.

Reply:
ok removed 👍